pyproject.toml

```diff
@@ -259,6 +259,7 @@ lint.flake8-implicit-str-concat.allow-multiline = false
 "tools".msg = "Use openpilot.tools"
 "pytest.main".msg = "pytest.main requires special handling that is easy to mess up!"
 "unittest".msg = "Use pytest"
+"pyray.measure_text_ex".msg = "Use openpilot.system.ui.lib.text_measure"
```

Comment:
why? I'd expect this to be explained in `openpilot.system.ui.lib.text_measure` but it isn't

Reply:
It caches `rl.measure_text_ex`. After viewing every text view in the raylib UI it only needs to calculate text sizes ~200 times. Without this caching, we need to run this function up to ~800 times per second depending on what is showing (this is toggles).

CPU drops from 11.6 to 8.6 showing the toggles on PC